services/agent/src/simcore_service_agent/core/settings.py

```diff
@@ -25,6 +25,11 @@ class ApplicationSettings(BaseCustomSettings, MixinLoggingSettings):
             "disabled if you want to have structured logs!"
         ),
     )
+    AGENT_VOLUMES_LOG_FILTER_MAPPING: dict = Field(
```

Comment:
I would recommend, since we have pydantic tightly integrated anyway, to use

1.) A saner layout for this dict than the scary, whitespace containing str(`'"GET / HTTP/1.1" 200"'`). I guess you are literally looking if the python logical condition `log-filter-string in logline == True`, which is a quick win. As mentioned before, I would right away go for regexes. Then, if e.g. the number of strings changes or so, we can quickly fix it with general regexes. Regexes in my mind are just the way to go for pattern matching, almost always

2. Dont just use a `dict[any,any]`, but restrict the content of the dict to valid loggers and valid regexes. Pydantic will validate this for you for free then.


Hearty suggestions but I wont break your balls over them.

Reply:
I do not want to restrict the code content to valid loggers because the purpose of this PR is to provide flexibility. Now, when we switch to INFO logs in production, we have the option to filter certain logs using an environment variable. This is a simple and straightforward approach. As mentioned before, this can be improved in the future if needed. I prefer not to 'overcomplicate' things that haven't been used yet.